src/main/scala/xiangshan/Parameters.scala

```diff
@@ -157,6 +157,7 @@ case class XSCoreParameters
     LduCnt = 2,
     StuCnt = 2
   ),
+  LsMaxRsDeq: Int = 4, // TODO: reduce it to 2 and delete it
```

Comment:
The maximun number of Load/Store RS's dequeue ports is not limitted to 2 yet. It is related to feedback logic. I will try to handle it soon.

Reply:
It has been deleted~~~